.github/workflows/tests.yml

```
@@ -1,35 +1,29 @@
----
-
 name: Tests
 
 on:
   push:
-    branches:
-      - master
   pull_request:
-    branches:
-      - master
 
 jobs:
   build_doc:
-    name: 'Build cpython doc'
+    name: Build CPython documentation
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: 3
       - name: clone docsbuild scripts
         run: |
           git clone https://github.com/python/docsbuild-scripts/
       - name: setup requirements
         run: |
           python -m pip install --upgrade pip
-          python -m pip install jinja2 markupsafe zc.lockfile
+          python -m pip install jinja2 markupsafe zc.lockfile requests
```

Let's use the existing requirements.txt to avoid duplication and needing to keep both in sync:

```suggestion
          python -m pip install -r docsbuild-scripts/requirements.txt
```